src/ugid.rs

```diff
@@ -86,6 +88,78 @@ impl Gid {
     }
 }
 
+impl fmt::Display for Uid {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)
+    }
+}
+impl fmt::Binary for Uid {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)
```

Comment:
Which `fmt` does this call? Does it call `Binary::fmt` or does it forward to `Display::fmt`?

And similar for all the other `fmt::*` impls.

Reply:
Always the right one

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=0748876dbb30e5dcc211d8d65ce1870b